src/interface/apiResponse.ts

```diff
@@ -23,6 +23,11 @@ export class FolderResponse {
   path: string;
 }
 
+export class DeleteFolderResponse {
+  @ApiProperty()
```

Comment:
Why do you set ApiProperty type string for DeleteFileRequest but don't set it for DeleteFolderResponse?